tests/authenticate.test.ts

```diff
@@ -0,0 +1,123 @@
+/* eslint-disable prettier/prettier */
+import axios from 'axios';
+import { Server } from 'http';
+import { readFileSync } from 'fs';
+import * as express from 'express';
+import { sign, verify } from 'jsonwebtoken';
+const configPath = `${__dirname}/config/continuum-auth.json`;
+const config = JSON.parse(readFileSync(configPath, 'utf8'));
+
+// port the mocked profile / redirect server will expose
+const MOCK_SERVER_PORT = 3002;
+// time in ms from now the mock journal token will be valid for
+const MOCK_TOKEN_EXP = 20000
+
+describe('Authenticate', (): void => {
+    let server: Server;
+    beforeAll(() => {
+        const app: express.Express = express();
+        app.get('/redirect_destination', (_, resp) => {
+            resp.status(200).send('Successful redirect');
+        });
+        app.get('/profiles/*', (_, resp) => {
+            resp.status(200).json({
+                id: 'TEST_ID',
+                name: {
+                    preferred: 'TEST_PREFERRED_NAME',
+                    index: 'TEST_INDEX_NAME',
+                },
+                orcid: '0000-0001-2345-6789',
+                emailAddresses: [
+                    {
+                        value: 'test@example.com'
+                    }
+                ],
+                affiliations: [],
+            });
+        });
+        server = app.listen(MOCK_SERVER_PORT);
```

Comment:
If this mock profile service is running locally, how will the continuum-auth service in the container access it?